src/utils/gaa-test.js

```diff
@@ -39,10 +40,14 @@ import {
 import {I18N_STRINGS} from '../i18n/strings';
 import {JwtHelper} from './jwt';
 import {tick} from '../../test/tick';
+import {resolve} from 'bluebird';
```

Comment:
Why do you need bluebird here? Node has native promises.

Reply:
Removed